caffe2/serialize/inline_container.cc

```diff
@@ -649,10 +676,22 @@ void PyTorchStreamWriter::setup(const string& file_name) {
       file_stream_.write(static_cast<const char*>(buf), nbytes);
       return !file_stream_ ? 0 : nbytes;
     };
+    seek_func_ = [this](size_t nbytes) -> size_t {
+      file_stream_.seekp(nbytes, std::ios_base::cur);
+      return !file_stream_ ? -1L : static_cast<size_t>(file_stream_.tellp());
```

Comment:
I'm a bit surprised by this conditions? If (!file_stream_), I guess the line above would have segfaulted? :p